src/features/locators/utils/utils.ts

```diff
@@ -35,6 +34,11 @@ export const evaluateCssSelector = (selector: string, element_id?: ElementId, or
     return response;
   });
 };
+export const generateSelectorByHash = (element_id: ElementId, jdnHash: string) => {
+  return sendMessage.generateSelectorByHash({ element_id, jdnHash }).then((response) => {
+    return response;
+  });
+};
```

Comment:
that's strange.. why do we need `return response`? `.then` looks almost useless. and in functions above too

Reply:
thank you!